app/src/main/java/com/pileproject/drive/execution/ExecutionThread.java

```diff
@@ -158,155 +117,56 @@ private void finalizeExecution() {
         }
     }
 
-    // Wait on the millisecond time scale
+    // wait in milliseconds
     private void waitMillSec(int milli) {
         try {
-            Thread.sleep(milli); // Sleep for milli sec
+            Thread.sleep(milli); // sleep for milli sec
         } catch (InterruptedException e) {
```

Comment:
it would be better to rethrow as non-check exception? same comment applies to other lines